lambda/db/src/migrations/2024.09.16T00.00.00.remove-vc-approved.ts

```diff
@@ -0,0 +1,17 @@
+import { DataTypes } from 'sequelize';
+
+export const name = '2024.09.16T00.00.00.remove-vc-approved';
+
+export const up = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().removeColumn('requests', 'digital_credential_approved');
+};
+
+export const down = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().addColumn('requests', 'digital_credential_approved', {
```

Comment:
Still required?

Reply:
yes its required for sequelize to remove the column